_data/products.yml

```diff
@@ -45,7 +45,7 @@
   category: productivity
   image: https://file.go.gov.sg/engagesg-logo.png
   sum: Build government forms in minutes
```

Comment:
Should update the summary as well 😅 . Thankfully I think it is not used anywhere atm...